Makefile

```
@@ -79,7 +79,7 @@ unit-test: ## Run unit-tests.
 ##@ Build
 
 manager: patch generate ## Build manager binary.
-	go build -mod=vendor -o /tmp/bin/manager main.go
+	go build -mod=vendor -o manager main.go
```

From `go help build`:
```
	-mod mode
		module download mode to use: readonly, vendor, or mod.
		By default, if a vendor directory is present and the go version in go.mod
		is 1.14 or higher, the go command acts as if -mod=vendor were set.
		Otherwise, the go command acts as if -mod=readonly were set.
		See https://golang.org/ref/mod#build-commands for details.
```
```suggestion
	go build -o manager main.go
```